### DISCUSSION TITLE
请问与此脚本是否有冲突

### DISCUSSION THREAD

**Discussion by fdjkdf** [the1812/Bilibili-Evolved#4139] [提问 / Q&A]
您好大佬，我一直在用您的Bilibili Evolved，但同时还想自由控制播放速度和前进后退，于是安装了如下两个HTML5播放器的脚本，但是发现同时一起和您的Bilibili Evolved运行时，两个HTML5有时候就会失效，不知道是哪里冲突了，为此很烦恼，希望能解答。

https://greasyfork.org/zh-CN/scripts/381682-html5%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E5%99%A8%E5%A2%9E%E5%BC%BA%E8%84%9A%E6%9C%AC

https://greasyfork.org/zh-CN/scripts/455396-html5%E6%92%AD%E6%94%BE%E5%99%A8%E8%87%AA%E5%AE%9A%E4%B9%89%E9%85%8D%E7%BD%AE

**Comment by Geeyun-JY3** [comment]
开了 Bilibili Evolved 的`快捷键扩展`应该就难免冲突。

**Reply by Geeyun-JY3** [reply]
今天试了一下，实测是这个 HTML5 脚本的快捷键优先生效（不确定和脚本的先后开关有没有关系）

**Reply by fdjkdf** [reply]
Bilibili Evolved 中 我没安装快捷键拓展，请问有其他什么功能会和它（HTML5脚本）冲突吗。  另外我用下来是 有时候它能生效 有时候不行，经常出现的场景是  看完一个视频后 点了右侧推荐的视频，会出现它不生效，此时Ctrl+R 也不行，必须此页面关掉（即Ctrl+W），再重新打开才可能正常生效。

我上面发了两个链接，除了安装HTML5的脚本，还有它的自定义配置脚本，我使用下来感觉如果开了那两个脚本和您的Bilibili Evolved后，那个自定义配置脚本貌似有时就会失效。

**Reply by Geeyun-JY3** [reply]
首先我不是 Bilibili Evolved 的开发者，也只是一名普通的用户。
其次你上述两个脚本我实测的时候都装了。

如果没装`快捷键扩展`那我也不清楚还有什么功能可能和这俩脚本冲突了，只能建议你先去逐个排查已装的其他 Bilibili Evolved 组件；还有就是再排除其他浏览器扩展或油猴脚本。

**Reply by the1812** [reply]
>  看完一个视频后 点了右侧推荐的视频，会出现它不生效

这个应该是 `网址参数清理` 的神秘问题, 我有遇到过但还不知道原因. (#4107)

其他有没有冲突我也不知道, 只能你自己开关功能试试
